netlify/functions/handler.js

```diff
@@ -0,0 +1,50 @@
+const { graphql } = require('@octokit/graphql')
```

Comment:
Nitpick: I think the name `handler.js` doesn't say anything about what this function does. I suggest we rename it to something meaningful. I.e. `github_discussions.js` or similar.

Reply:
I will add comments above the function to explain the functionality of the function in brief.